hoomd/md/minimize/fire.py

```diff
@@ -0,0 +1,283 @@
+            forces are summed together. The default value of ``None``
+            initializes an empty list.
+        methods (Sequence[hoomd.md.methods.Method]):
+            Sequence of integration methods. Each integration method can be
+            applied to only a specific subset of particles. The intersection of
+            the subsets must be null. The default value of ``None`` initializes
+            an empty list.
+        constraints (Sequence[hoomd.md.constrain.Constraint]):
+            Sequence of constraint forces applied to the particles in the
+            system. The default value of ``None`` initializes an empty list.
+            Rigid body objects (i.e. `hoomd.md.constrain.Rigid`) are not
+            allowed in the list.
+        rigid (hoomd.md.constrain.Rigid):
+            A rigid bodies object defining the rigid bodies in the simulation.
+        min_steps_adapt (int):
+            Number of steps energy change is negative before allowing
+            :math:`\\alpha` and :math:`\\delta t` to adapt.
+        finc_dt (float):
+            Factor to increase :math:`\\delta t` by
+            :math:`[\\mathrm{dimensionless}]`.
+        fdec_dt (float):
+            Factor to decrease :math:`\\delta t` by
+            :math:`[\\mathrm{dimensionless}]`.
+        alpha_start (float):
+            Initial (and maximum) :math:`\\alpha [\\mathrm{dimensionless}]`.
+        fdec_alpha (float):
+            Factor to decrease :math:`\\alpha t` by
+            :math:`[\\mathrm{dimensionless}]`.
+        force_tol (float):
+            Force convergence criteria
+            :math:`[\\mathrm{force} / \\mathrm{mass}]`.
+        angmom_tol (float):
+            Angular momentum convergence criteria
+            :math:`[\\mathrm{energy} * \\mathrm{time}]`.
+        energy_tol (float):
+            Energy convergence criteria :math:`[\\mathrm{energy}]`.
+        min_steps_conv (int):
+            A minimum number of attempts before convergence criteria are
+            considered.
+
+    `minimize.FIRE` is an `md.Integrator` that uses the Fast Inertial Relaxation
+    Engine (FIRE) algorithm to minimize the potential energy for a group of
+    particles while keeping all other particles fixed. This method is published
+    in `Bitzek, et. al., PRL, 2006
+    <http://dx.doi.org/10.1103/PhysRevLett.97.170201>`_.
+
+    At each time step, :math:`\\delta t`, the algorithm uses the supplied
+    integration methods to generate a x, v, and F, and then adjusts v according
+    to
+
+    .. math::
+
+        \\vec{v} = (1-\\alpha)\\vec{v} + \\alpha \\hat{F}|\\vec{v}|
+
+    where :math:`\\alpha` and :math:`\\delta t` are dynamically adaptive
+    quantities.  While a current search has been lowering the energy of system
+    for more than :math:`N_{min}` steps, :math:`\\alpha` is decreased by
+    :math:`\\alpha \\rightarrow \\alpha fdec_{\\alpha}` and :math:`\\delta t`
```

Comment:
```suggestion
    :math:`\\alpha \\rightarrow \\alpha \\mathrm{fdec}_{\\alpha}` and :math:`\\delta t`
```
And similarly where multi-character variables are used (fdec, finc, etc...). Without mathrm, these appear as the multiplication of f, d, e, and c.